resources/assets/less/beatmaps.less

```diff
@@ -226,9 +226,7 @@
 	margin: 5px;
 
 	&:hover {
-		z-index: 500;
-		/*.stats { color: @pink; }*/
-
+		z-index: 4;
```

Comment:
z-index goes in the variables file (also for all other z-indexes not using variable) :fireworks: